### PR TITLE
Improve docs of softplus

### PR DESCRIPTION
**Pull request by keisuke-umezawa**
This is PR for improving docs of functions/links. Related issue: #2182

Modified functions/links:
- functions/activation/softplus

### CODE REVIEW

File: chainer/functions/activation/softplus.py [hunk 1]
```diff
@@ -58,18 +58,27 @@ def backward_gpu(self, inputs, grads):
 
 
 def softplus(x, beta=1.0):
-    """Elementwise softplus function.
+    """Element-wise softplus function.
 
-    This function is expressed as
-    :math:`f(x) = \\frac{1}{\\beta}\\log(1 + \\exp(\\beta x))`,
-    where :math:`\\beta` is a parameter.
+    .. math:: f(x)=\\frac{1}{\\beta}\\log(1 + \\exp(\\beta x)).
```
Comment (mitmul) [inline chainer/functions/activation/softplus.py]: Please add a brief explanation of this function here. SoftPlus is a smooth approximation to the ReLU, so the output value is always positive. And how the output value will change when the `beta` changed?

File: chainer/functions/activation/softplus.py [hunk 2]
```diff
@@ -58,18 +58,27 @@ def backward_gpu(self, inputs, grads):
...
```
Comment (mitmul) [inline chainer/functions/activation/softplus.py]: Please add a notation for `beta`, for example:
where `beta` denotes a parameter.

**Comment by keisuke-umezawa** [comment]
I fixed the docstrings as mentioned above. Please merge after the test is passed.

**Comment by keisuke-umezawa** [comment]
I checked doctest and it's okay.
```
...

Document: reference/functions
-----------------------------
1 items passed all tests:
  87 tests in default
87 tests in 1 items.
87 passed and 0 failed.
Test passed.

...

Doctest summary
===============
  308 tests
    8 failures in tests
    0 failures in setup code
    0 failures in cleanup code
build finished with problems, 9 warnings.
make: *** [doctest] Error 1
```

**Comment by keisuke-umezawa** [comment]
I rebased the branch against master. pls rerun the Jenkins test.

**Comment by mitmul** [comment]
LGTM

**Pull request merged**